strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/artifact/generator/MavenArtifactGenerator.java

```diff
@@ -264,13 +282,14 @@ private void createMavenPropertiesFile(Artifact artifact, ZipOutputStream zos)
         zos.closeEntry();
     }
 
-    private void createRandomSizeFile(ZipOutputStream zos)
+    private void createFile(ZipOutputStream zos,
+                            long size)
             throws IOException
     {
-        ZipEntry ze = new ZipEntry("random-size-file");
```

Comment:
I think you should add an if check here... If a size hasn't been specified, create the random-sized file (with the limit as it was previously).

Reply:
In all cases where size is not specified it is randomed - for example in `public void generate(Artifact artifact)`. I will change that random limit to previous value (1000000).